lib/ansible/modules/network/skydive/skydive_node.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: skydive_node
+version_added: "2.8"
+author:
+  - "Sumit Jaiswal (@sjaiswal)"
+short_description: Module which add nodes to Skydive topology
+description:
+  - This module handles adding node to the Skydive topology. The Gremlin
+    expression is continuously evaluated which means that it is
+    possible to define a capture on nodes that do not exist yet.
+  - It is useful when you want to start a capture on all OpenvSwitch
+    whatever the number of Skydive agents you will start.
+  - While starting the capture, user can specify the capture name,
+    capture description and capture type optionally.
+requirements:
+  - skydive-client
+extends_documentation_fragment: skydive
+options:
+  name:
+    description:
+      - To define name for the node.
+    required: true
+  node_type:
```

Comment:
I am not sure you should rename skydive parameters. May be NodeType would have been a better choice than Type in Skydive, may be RelationType could have been replaced by Type. But users of this module are Skydive users. They expect a name like type or Type, not node_type. Having different names for the same concept in different settings is misleading.

Reply:
I had changed the `type` to `node_type` coz `type` is python built-in keyword, if you want I can change the `name` to `node_name` to be in conjunction with Skydive UI where it's `Node Name` and `Node Type` respectively.

Reply:
You can use 'type' as a dictionary entry field. Ansible uses 'type' as a dict entry for module spec (you use it for the edge module ibb_spec). Note that if you insist on node_type, you must modify the examples that still refer to 'type'.

Reply:
I have updated the `skydive_edge` module example to `node_type`.